operations/loyaltymemberships.md

```diff
@@ -0,0 +1,280 @@
+# Loyalty memberships
+
+## Get all loyalty memberships
+
+Returns all loyalty memberships of the enterprise (in the given activity state), optionally filtered by specific loyalty membership identifiers or other filter parameters.
```

Comment:
technically "in the given activity states" (plural)

Reply:
Fixed 👍🏻